frontend/tablet/src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { StyleSheet, Text } from 'react-native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+
+interface Props {
```

Comment:
Let's use `type` instead of `interface` for consistency. Using names such as `ButtonProps` gives advantages in more complicated components where you can have multiple `Props` in one file. It also improves the TypeScript experience when you look at some component with a mouse hover.